test/integration_test/user_defined/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -u
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $cur
+
+bin_path=../../../bin
+
+# test user defined attack
+${bin_path}/chaosd attack user-defined --attack-cmd "touch /tmp/chaos" --recover-cmd "rm /tmp/chaos" --uid 1234567890
+
+if [ ! -f /tmp/chaos ]; then
```

Comment:
If other progress just created a directory called "/tmp/chaos", then "touch /tmp/chaos" will not report an error, but [ ! -f /tmp/chaos ] will get true, so is it possible to use '- e' would be more appropriate?

Reply:
done